srcpkgs/python3-adblock/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'python3-adblock'
+pkgname=python3-adblock
+version=0.4.2
+revision=1
+wrksrc="${pkgname/python3/python}-${version}"
+build_style=python3-pep517
+build_helper="rust"
+hostmakedepends="maturin pkg-config cargo"
+makedepends="libressl-devel python3-devel"
+depends="python3"
+checkdepends="python3-pytest"
+short_desc="Brave's adblock library in Python"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="Apache-2.0, MIT"
+homepage="https://github.com/ArniDagur/python-adblock"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=06de6074e6cfe889fc0383cc929a5a2306570251c14e51abbfcedd328b83e0e9
+nocross="openssl-sys gets wrong compiler flags on cross targets"
+
+case "$XBPS_TARGET_MACHINE" in
+	i686*) broken="compiler throws SIGABRT on the psl crate" ;;
+esac
+
+if [ "$CROSS_BUILD" ]; then
+	makedepends+=" rust-std"
+	export PYO3_CROSS_LIB_DIR="${XBPS_CROSS_BASE}/usr/lib"
+	export PYO3_CROSS_INCLUDE_DIR="${XBPS_CROSS_BASE}/usr/include"
+fi
+
+do_build() {
+	if [ "$XBPS_TARGET_LIBC" = musl ]; then
+		RUSTFLAGS+=" -C target-feature=-crt-static"
+	fi
```

Comment:
Does this make it link in static openssl?

Reply:
No idea, I cargo culted that from the older PRs. You can see if it builds without that feature if you like.

Reply:
Ah, ok.

Reply:
Seems to build and work just as well without that...

Reply:
Though it's still not linked against libssl :/